### DISCUSSION TITLE
How to omit fields from GraphQL schema?

### DISCUSSION THREAD

**Discussion by GV14982** [ent/ent#3515] [Q&A]
So I'm using entgql to generate graphql code, but I have some fields in the database that I want to omit from the generated graphql schema. Is there an easy way to do this?

**Comment by a8m** [comment]
Hey, @GV14982 👋🏻
You can use the `entgql.Skip` annotation. See the examples in: https://github.com/ent/contrib/blob/master/entgql/internal/todo/ent/schema/todo.go

**Comment by a8m** [comment]
Feel free to reopen the discussion if you need more help with this 🙂
